bundle/Resources/config/services.yaml

```diff
@@ -11,3 +11,14 @@ services:
 
     netgen_open_graph.meta_tag_renderer:
         class: Netgen\Bundle\OpenGraphBundle\MetaTag\Renderer
+
+    app.opengraph.handler.url:
```

Comment:
There's `handlers.yaml` that defines all handlers. This needs to go there. Also needs to be named appropriately `netgen_open_graph.handler.literal.canonical_url`